sorc/link_workflow.sh

```diff
@@ -65,7 +70,13 @@ ${LINK_OR_COPY} "${HOMEgfs}/versions/run.${machine}.ver" "${HOMEgfs}/versions/ru
 #------------------------------
 case "${machine}" in
   "wcoss2")   FIX_DIR="/lfs/h2/emc/global/noscrub/emc.global/FIX/fix" ;;
-  "hera")     FIX_DIR="/scratch1/NCEPDEV/global/glopara/fix" ;;
+  "hera")     
+    if [[ "${LINK_NEST:-OFF}" == "ON" ]] ; then
+      FIX_DIR="/scratch2/BMC/wrfruc/Guoqing.Ge/fix"
```

Comment:
This is a temporary change to bring awareness.
Once the fix files are updated as requested in https://github.com/NOAA-EMC/global-workflow/issues/2529, this will be removed.

Reply:
To be specific, only changes from line 73-79 are temporary in this PR.